app/components/data/home/data-home.component.ts

```diff
@@ -57,4 +72,8 @@ export class DataHomeComponent implements OnDestroy {
     private _updateFilter() {
         this.filter = this.quickFilter;
     }
+
+    private _createEmptyFileGroup(name: string) {
+        return this.storageService.createContainer(name);
+    }
 }
```

Comment:
does this prefix the container with fgrp-?

Reply:
yeah, as storage is crashing right now couldn't check it was working